altapay.php

```diff
@@ -0,0 +1,2390 @@
+                                'name'      => 'Authorize and capture'
+                            ],
+                        ],
+                        'id'    => 'id_option',
+                        'name'  => 'name'
+                    ]
+                ],
+                [
+                    'type'     => 'radio',
+                    'label'    => $this->l('Status'),
+                    'name'     => 'active',
+                    'required' => true,
+                    'is_bool'  => true,
+                    'values'   => [
+                        [
+                            'id'    => 'active_on',
+                            'value' => 1,
+                            'label' => $this->l('Enabled')
+                        ],
+                        [
+                            'id'    => 'active_off',
+                            'value' => 0,
+                            'label' => $this->l('Disabled')
+                        ],
+                    ],
+                ],
+            ],
+            'submit'  => [
+                'title' => $this->l('Save'),
+            ],
+            'buttons' => [
+                [
+                    'href'  => AdminController::$currentIndex .
+                               '&configure=' . $this->name . '&token=' . Tools::getAdminTokenLite('AdminModules'),
+                    'title' => $this->l('Back to list'),
+                    'icon'  => 'process-icon-back'
+                ]
+            ],
+        ];
+        $helper                        = new HelperForm();
+        $helper->module                = $this;
+        $helper->name_controller       = 'altapay';
+        $helper->identifier            = $this->identifier;
+        $helper->token                 = Tools::getAdminTokenLite('AdminModules');
+        $helper->currentIndex          = AdminController::$currentIndex . '&configure=' . $this->name;
+        $helper->show_toolbar          = false;
+        $helper->table                 = 'altapay_terminals';
+        $lang                          = new Language((int)Configuration::get('PS_LANG_DEFAULT'));
+        $helper->default_form_language = $lang->id;
+        $helper->id                    = (int)Tools::getValue('id_terminal');
+        $helper->submit_action         = 'savealtapay_terminals';
+        $helper->tpl_vars              = [
+            'fields_value' => (array)$this->getFormValues(),
+            'languages'    => (array)$this->context->controller->getLanguages(),
+            'id_language'  => (array)$this->context->language->id
+        ];
+
+        return $helper->generateForm($fieldsForm);
+    }
+
+    /**
+     * Query the AltaPay API for available terminals
+     *
+     * @param bool $objects
+     *
+     * @return array<int, Terminal>
+     */
+    private function getAltapayTerminals($objects = false)
+    {
+        require_once(_PS_MODULE_DIR_ . '/altapay/lib/AltapayMerchantAPI.class.php');
```

Comment:
About 4 of thease left
```suggestion
        require_once _PS_MODULE_DIR_ . '/altapay/lib/AltapayMerchantAPI.class.php';
```